src/buildercore/utils.py

```diff
@@ -65,6 +65,14 @@ def wrapper(*args, **kwargs):
 def splitfilter(func, data):
     return filter(func, data), filter(complement(func), data)
 
+def mkidx(fn, lst):
```

Comment:
is there a Pylint check for too low percentage of vowels in a row? :nerd_face:

Reply:
> too low percentage of vowels in a row

probably, but I also don't consider it a problem. we should spend more time addressing low test coverage or low quality tests than linting. linting after a certain point is just nest feathering and self gratification.